packages/uniforms-material/src/ErrorsField.tsx

```diff
@@ -1,8 +1,9 @@
-import { useTheme, PropTypes } from '@material-ui/core';
+import type { PropTypes } from '@material-ui/core';
 import FormControl from '@material-ui/core/FormControl';
 import FormHelperText, {
   FormHelperTextProps,
 } from '@material-ui/core/FormHelperText';
+import useTheme from '@material-ui/core/styles/useTheme';
```

Comment:
Is it faster?

Reply:
Nope, but it got rid of [this warning](https://github.com/vazco/uniforms/actions/runs/4384694127/jobs/7676475057#step:8:16).